src/app/services/config/localforageconfig.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { NgForageConfig} from 'ngforage';
+import { ONESKY , ONESKYSTORAGE} from '../constants';
+@Injectable()
+export class LocalforageconfigService {
```

Comment:
Use CamelCase notations replace `LocalforageconfigService ` with `LocalforageConfigService `
And also replace where it is imported